js/src/carousel.js

```diff
@@ -390,10 +390,12 @@ const Carousel = (($) => {
 
         if (typeof config === 'number') {
           data.to(config)
-
-        } else if (action) {
-          data[action]()
-
+        } else if (typeof action === 'string') {
+          try {
```

Comment:
yes it's a better solution ! Thank you !
I'll generalize this check for all plugins